src/utils/types/customerConstants.ts

```diff
@@ -24,3 +24,24 @@ export const FORM_FIELDS = [
   { label: "Confirm Password", type: "password", name: "confirm_password" },
   { label: "location", type: "number", name: "location_id" },
 ] as const;
+
+export const Locations: Location[] = [
+  { id: 1, country: "United Kingdom", region: "North East" },
+  { id: 2, country: "United Kingdom", region: "North West" },
+  { id: 3, country: "United Kingdom", region: "Yorkshire and the Humber" },
+  { id: 4, country: "United Kingdom", region: "East Midlands" },
+  { id: 5, country: "United Kingdom", region: "West Midlands" },
+  { id: 6, country: "United Kingdom", region: "East of England" },
+  { id: 7, country: "United Kingdom", region: "London" },
+  { id: 8, country: "United Kingdom", region: "South East" },
+  { id: 9, country: "United Kingdom", region: "South West" },
+  { id: 10, country: "France" }, // Remove region property entirely for countries without regions
+  { id: 11, country: "Spain" },
+  { id: 12, country: "Italy" },
+  { id: 13, country: "Germany" },
+  { id: 14, country: "Sweden" },
+  { id: 15, country: "Denmark" },
+  { id: 16, country: "Netherlands" },
+  { id: 17, country: "Norway" },
+  { id: 18, country: "Croatia" },
+];
```

Comment:
I don't think these should be hardcoded they should be fetched from the database

Reply:
I will get on this after I make some of the changes we discussed in our stand up. This plus the genre covers aer a bit difficult for me to complete without going over our documentation